glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -176,18 +183,36 @@ int[] GetMaterials(RenderMaterial[] materials, RhinoObject rhinoObject)
                     return null;
```

Comment:
This check should probably happen before GetMaterials gets called. At the very least it should be outside the for loop.